ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -213,15 +213,19 @@ conformance_test(
         "//ledger/test-common/test-certificates:client.pem",
         "//ledger/test-common/test-certificates:server.crt",
         "//ledger/test-common/test-certificates:server.pem",
+        "//ledger/test-common/test-certificates:server.pem.enc",
         "//ledger/test-common/test-certificates:ca.crt",
+        "//ledger/test-common/test-certificates:server-pem-decryption-parameters.json",
     ],
     server = "//ledger/ledger-on-memory:app",
     server_args = [
         "--contract-id-seeding=testing-weak",
         "--participant participant-id=ssl-test,port=6865",
         "--crt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.crt))",
         "--cacrt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:ca.crt))",
-        "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem))",
+        "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem.enc))",
+        # TODO PBATKO: Is prepending `file://` platoform independent? Windows?
```

Comment:
The [file URI protocol](https://datatracker.ietf.org/doc/html/rfc8089) is platform independent, but I would try to use https URLs whenever possible. How about a gist, or `https://raw.githubusercontent.com/digital-asset/daml/main/<path of a file within our repo>`?

Reply:
Ok